server/src/main/java/org/candlepin/async/JobManager.java

```diff
@@ -1293,6 +1295,9 @@ private void setupJobRuntimeEnvironment(AsyncJobStatus status) {
         MDC.put(MDC_REQUEST_TYPE_KEY, "job");
         MDC.put(MDC_REQUEST_UUID_KEY, status.getId());
 
+        MDC.put(MDC_REQUEST_SUB_TYPE_KEY, "jobKey");
```

Comment:
For the displayed value, we want to use snake case, so "job_key" here is more appropriate.

Reply:
Dropped this key from here.